src/tools/blast/blastcommandline.jl

```diff
@@ -0,0 +1,49 @@
+# wrapper for BLAST+ command line functions
+
+"""
+`blastn(query, subject, flags...)``
+Runs blastn on `query` against `subject`. Query may be string or DNASequence.
+    Subjects and queries may be file names (as strings), DNASequence type or
+    Array of DNASequence.
+    May include optional `flags` such as `[-pident, 95]`. Do not use `-outfmt`.
+"""
+function blastn(query::ASCIIString, subject::ASCIIString, flags...)
+    result = readall(`blastn -query $query -subject $subject -outfmt 6 $flags`)
+    # do stuff
+end
+
+function blastn(query::DNASequence, subject::DNASequence, flags...)
+    querypath, subjectpath = makefasta(query), makefasta(subject)
+    result = readall(`blastn -query $querypath -subject $subjectpath -outfmt 6 $flags`)
```

Comment:
Would it make sense here simply call `blastn(querypath, subjectpath)`, which would use the version of the function that operates on `ASCIIString`s? Can't tell if it's super unclear to call a function of the same name (but that acts on different types) within a function.